plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/snowflake/TestIcebergSnowflakeCatalogConnectorSmokeTest.java

```diff
@@ -0,0 +1,768 @@
+                    SUPPORTS_MERGE,
+                    SUPPORTS_UPDATE,
+                    SUPPORTS_RENAME_TABLE,
+                    SUPPORTS_ROW_LEVEL_UPDATE,
+                    SUPPORTS_ROW_LEVEL_DELETE,
+                    SUPPORTS_CREATE_OR_REPLACE_TABLE,
+                    SUPPORTS_CREATE_TABLE_WITH_DATA,
+                    SUPPORTS_COMMENT_ON_TABLE,
+                    SUPPORTS_COMMENT_ON_COLUMN,
+                    SUPPORTS_COMMENT_ON_VIEW -> false;
+            default -> super.hasBehavior(connectorBehavior);
+        };
+    }
+
+    @Test
+    @Override
+    public void testView()
+    {
+        assertThatThrownBy(super::testView)
+                .hasStackTraceContaining("This connector does not support creating views");
+    }
+
+    @Test
+    @Override
+    public void testMaterializedView()
+    {
+        assertThatThrownBy(super::testMaterializedView)
+                .hasStackTraceContaining("This connector does not support creating materialized views");
+    }
+
+    @Test
+    @Override
+    public void testRenameSchema()
+    {
+        assertThatThrownBy(super::testRenameSchema)
+                .hasStackTraceContaining("This connector does not support renaming schemas");
+    }
+
+    @Test
+    @Override
+    public void testRenameTable()
+    {
+        assertThatThrownBy(super::testRenameTable)
+                .hasStackTraceContaining("This connector does not support renaming tables");
+    }
+
+    // Overridden as the table location and column data type is different
+    @Test
+    @Override
+    public void testShowCreateTable()
+    {
+        Function<ResultSet, String> descExternalVolumeOperator = resultSet -> {
+            try {
+                while (resultSet.next()) {
+                    if (resultSet.getString("parent_property").equals("STORAGE_LOCATIONS")
+                            && resultSet.getString("property").equals("STORAGE_LOCATION_1")) {
+                        return resultSet.getString("property_value");
+                    }
+                }
+                throw new RuntimeException("Snowflake external volume's required property is missing");
+            }
+            catch (SQLException e) {
+                throw new RuntimeException(e);
+            }
+        };
+
+        Optional<String> storageLocation;
+        try {
+            storageLocation = server.executeOnDatabaseWithResultSetOperator(SNOWFLAKE_TEST_SCHEMA, Optional.of(descExternalVolumeOperator), "DESC EXTERNAL VOLUME %s".formatted(SNOWFLAKE_S3_EXTERNAL_VOLUME));
+        }
+        catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+
+        Optional<String> baseUrl = storageLocation.map(location -> {
+            try {
+                return OBJECT_MAPPER.readTree(location).get("STORAGE_BASE_URL").asText();
+            }
+            catch (JsonProcessingException e) {
+                throw new RuntimeException(e);
+            }
+        });
```

Comment:
Assertion with pattern instead of exact location is acceptable for me. Do we have any reason for verifying the exact location?

Reply:
It was suggested through https://github.com/trinodb/trino/pull/21365#discussion_r1575443739

Reply:
That's a valid request unless it introduces complex logic.

Reply:
Makes sense. Changed it to use the regex pattern.